src/pages/CartPage/components/CartItemList.tsx

```
@@ -0,0 +1,66 @@
+            />
+          )}
+          <div className="flex-1">
+            <h3 className="font-medium text-slate-900">{item.name}</h3>
+            <p className="text-slate-600">${item.price.toFixed(2)}</p>
+          </div>
+          <div className="flex items-center gap-2">
+            <button
+              onClick={() => onUpdateQuantity(item.id, item.quantity - 1)}
+              className="flex h-8 w-8 items-center justify-center rounded-full border border-slate-300 hover:bg-slate-50"
+              disabled={item.quantity <= 1}
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)